src/Component.php

```diff
@@ -104,6 +104,11 @@ protected function runSampleAction(): array
         $profile = $this->getConfig()->getProfiles()[0];
         $parameters = $this->getConfig()->getParameters();
 
+        if ($this->getConfigDefinitionClass() === OldConfigDefinition::class) {
+            $parameters += $parameters['queries'][0];
```

Comment:
vždy to je jen na jednu query - je to akce při vytváření konfigurace na ukázkový data z GA